bin/setup

```diff
@@ -6,7 +6,7 @@ def setup!
     run  "gem install bundler --no-document --conservative"
     run  "bundle install"
     run  "bin/yarn install" if File.exist?("yarn.lock")
-    run  "bundle exec overcommit --install"
+    run  "bundle exec overcommit --install" unless ENV.include? "SKIP_INSTALLING_OVERCOMMIT_IN_SETUP"
```

Comment:
:elephant: this _could_ be a helper method - `run "bundle exec overcommit --install" if install_overcommit?`.